Obok_plugin/obok/obok.py

```diff
@@ -449,9 +449,15 @@ def __getmacaddrs (self):
             for m in matches:
                 # print "m:{0}".format(m[0])
                 macaddrs.append(m[0].upper())
+        elif sys.platform.startswith('linux'):
+            for interface in os.listdir('/sys/class/net'):
+                with open('/sys/class/net/' + interface + '/address', 'r') as f:
```

Comment:
I used string concat here instead of f-strings because I wasn't sure if I could assume at least Python 3.6.